src/ui/screens/Parachains/ParathreadsScreen.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {ReactTestInstance} from 'react-test-renderer';
+import {render, waitFor, within} from 'src/testUtils';
+import {ParathreadsScreen} from './ParathreadsScreen';
+
+describe('OverviewScreen component', () => {
+  it('should render the loading view when data is fetching', async () => {
+    const {getByTestId} = render(<ParathreadsScreen />);
+    expect(getByTestId('loading_view')).toBeTruthy();
+  });
+
+  it('When data is rendered', async () => {
```

Comment:
Let's use a more descriptive text for the test. it should make sense when you read "it should/must/etc.... what you expect from this case"